pkg/types/profit.go

```diff
@@ -182,6 +185,8 @@ func NewProfitStats(market Market) *ProfitStats {
 		TodayGrossProfit:       fixedpoint.Zero,
 		TodayGrossLoss:         fixedpoint.Zero,
 		TodaySince:             0,
+		startTime:              time.Now().UTC(),
+		endTime:                time.Now().UTC(),
```

Comment:
could be errored if you're running backtesting.

Reply:
ah! so how do i get the "now" in backtesting?

Reply:
interesting, I guess we have the time in the bbgo.Environment, if not, we will need to add it LOL

Reply:
nope, we don't have "now" in bbgo.Environment. we have StartTime() only. I just commented out those lines related to time. will add them back once we have "now" in bbgo.Environment